src/go/MONIT/ggus_alerting.go

```diff
@@ -0,0 +1,357 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"os"
+	"strconv"
+	"time"
+)
+
+// File       : ggus_alerting.go
+// Author     : Rahul Indra <indrarahul2013 AT gmail dot com>
+// Created    : Thu, 21 May 2020 21:07:32 GMT
+// Description: GGUS Alerting Module for CERN MONIT infrastructure
+
+//URL for AlertManager
+var alertManagerURL string
+
+//severity of alerts
+var severity string
+
+//GGUS tag
+var tag string
+
+//verbose defines verbosity level
+var verbose int
+
+//Map for storing Existing Tickets
+var exstTkt map[string]int
+
+//GGUS Data Struct
+type ggus []struct {
+	TicketID        int    `json:"TicketID"`
+	Type            string `json:"Type"`
+	VO              string `json:"VO"`
+	Site            string `json:"Site"`
+	Priority        string `json:"Priority"`
+	ResponsibleUnit string `json:"ResponsibleUnit"`
+	Status          string `json:"Status"`
+	LastUpdate      int    `json:"LastUpdate"`
+	Subject         string `json:"Subject"`
+	Scope           string `json:"Scope"`
+}
+
+//AlertManager API acceptable JSON Data for GGUS Data
+type amJSON struct {
+	Labels struct {
+		Alertname string `json:"alertname"`
+		Severity  string `json:"severity"`
+		Tag       string `json:"tag"`
+	} `json:"labels"`
+	Annotations struct {
```

Comment:
Similar here, the Annotations is identical in both structs, in amJSON and in getAMJSON. You should define it outside of them and then use it.

Reply:
Fixed the getAMJSON struct. Named to ggusData, removed unused variables.